sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -314,9 +314,9 @@ public void Upsert_And_Retrieve_Simple_Orders()
 
             var t = upsertResult.Values.First().Version.AsAtDate;
 
-            var order1Filter = $"{testScope}/{order1}";
-            var order2Filter = $"{testScope}/{order2}";
-            var order3Filter = $"{testScope}/{order3}";
+            var order1Filter = $"{order1}";
+            var order2Filter = $"{order2}";
+            var order3Filter = $"{order3}";
```

Comment:
Given that `order1` etc are already strings, we could just pass them directly into the filter on line 323 - might be a bit clearer?